src/components/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { includes } from 'ramda';
+
+import { BreadcrumbItemProps } from './Breadcrumbs.types';
+import { getColor, getFontWeight } from '../../utils/helpers';
+import { LinkProps } from '../typography/Link/Link.types';
+import { Link } from '../typography/Link';
+
+const BreadcrumbLink = styled(Link).withConfig<
+  LinkProps & { isSelected: boolean }
+>({
+  shouldForwardProp: (property) => !includes(property, ['$isSelected']),
+})`
+  font-weight: ${getFontWeight('regular')};
+    &,
+    &:not([href]):not([tabindex]) {
+      color: ${({ $isSelected }) =>
+        $isSelected ? getColor('neutral.700') : getColor('neutral.900')}; };
```

Comment:
This one is shorter :)
```suggestion
         getColor($isSelected ? 'neutral.700' : 'neutral.900')};
```